web/pages/index.md

```diff
@@ -41,6 +40,7 @@ call/cc.
 
 ## Resources
 
+- A set of <a href="/k-distribution/pl-tutorial/README.md">reference implementations and tutorials</a> for common programming language features and paradigms is available, although parts of these implementations may not be fully up to date with modern K features.
```

Comment:
Could we mention somewhere which definitions are not up to date?
We may be able to tell by looking at the Makefiles, but someone from outside won't.
The best solution I can think of right now is to add a note at the beginning of each file:
```
// NOTE: this definition is not up to date with the latest version of K.
// It is here for documentation/academic purposes.
```

Reply:
Yeah, the only way to tell what's up to date is to look at what _actually_ gets run in CI via the Makefile groups. I'll go through and add something like that message to the ones that don't get run in CI.